lib/connection/connection.js

```diff
@@ -7,9 +7,15 @@ var inherits = require('util').inherits
   , crypto = require('crypto')
   , f = require('util').format
   , debugOptions = require('./utils').debugOptions
+  , parseHeader = require('./utils').parseHeader
+  , compressorIDs = require('./utils').compressorIDs
   , Response = require('./commands').Response
   , MongoError = require('../error')
-  , Logger = require('./logger');
+  , Logger = require('./logger')
+  , zlib = require('zlib')
+  , Snappy = require('./utils').retrieveSnappy()
+  , OP_COMPRESSED = require('./commands').OP_COMPRESSED
+  , MESSAGE_HEADER_SIZE = require('../wireprotocol/shared').MESSAGE_HEADER_SIZE;
```

Comment:
it kind of feels like all of the op constants should live in `wireprotocol/shared`. e.g. `OP_COMPRESSED`, `OP_QUERY`, etc